community/security/src/test/java/org/neo4j/server/security/auth/BasicAuthManagerTest.java

```diff
@@ -59,18 +59,18 @@ public void shouldFindAndAuthenticateUserSuccessfully() throws Throwable
     {
         // Given
         final InMemoryUserRepository users = new InMemoryUserRepository();
-        final User user = new User( "jake", Credential.forPassword( "abc123" ), false );
+            final User user = new User( "jake", "admin", Credential.forPassword( "abc123" ), false );
```

Comment:
Since this is the community auth manager, we could also argue that there is no concept of groups here.